### ISSUE TITLE
mt5-base model (unicamp-dl/mt5-base-en-msmarco) throws 'nan' outputs

### ISSUE THREAD

**Issue by cramraj8** [unicamp-dl/mMARCO#20]
Hi, 

I was running `unicamp-dl/mt5-base-en-msmarco`: ['▁no'   , '▁yes'] model for both English and other My.TyDi languages, but the output scores are `nan`. When I switched to `unicamp-dl/mt5-13b-mmarco-100k`: ['▁', '▁true'] model, I get actual logits. I wonder if there's any issues with underlying `unicamp-dl/mt5-base-en-msmarco` model.

Thanks.

**Comment by rodrigonogueira4** [comment]
Hi @vjeronymo2 @lhbonifacio any ideas of what could be the problem here?

**Comment by lhbonifacio** [comment]
Hey @cramraj8,
Thank you for your interest in our work.
Regarding `unicamp-dl/mt5-13b-mmarco-100k`, the prediction tokens are indeed `['▁', '▁true']` (as reported [here](https://github.com/castorini/pygaggle/blob/08339dd31f58ef40fbaa109726402e164eeba125/pygaggle/rerank/transformer.py#L54)).
Maybe @vjeronymo2 can give us more details about that, but as you already mentioned, it is working fine.

For the `unicamp-dl/mt5-base-en-msmarco` model, I just tested it using the reranking implementation from [InPars](https://github.com/zetaalphavector/InPars) and it seems to be working fine with the prediction tokens `['▁no'   , '▁yes']`. 
Maybe you could try to use your reranking code that is available here: https://github.com/zetaalphavector/InPars/blob/master/inpars/rerank.py 

Thanks.
